content/en/dashboards/_index.md

```diff
@@ -107,7 +107,7 @@ Use the pop up to restrict access to you, everyone in your organization with you
 
 Creators are always able to edit the dashboard, but other users who are allowed to edit the dashboard can add or remove any role from the access control list (ACL) so long as the final ACL includes one of their roles. For more information about roles, see the [RBAC documentation][10].
 
-If you used the deprecated "read only" check box, the access control list is pre-populated with a list of Admin roles with the "Privileged Access" permission.
+If the dashboard was created with the deprecated "read only" setting, the access control list pre-populates with a list of roles with the User Access Manage permission.
```

Comment:
Discussed this with you on our call today but I think we should refer to the permission as `user_access_manage`.

Or use both the internal and display name like "Access Management (`user_access_manage`)" or "`user_access_manage` (Access Management)"

Reply:
✅  I double-checked that the described behavior is accurate